bitnami/kafka/README.md

```diff
@@ -122,6 +122,12 @@ The following tables lists the configurable parameters of the Kafka chart and th
 | `service.nodePort`                      | Kubernetes Service nodePort                                                                                                                               | `nil`                                                   |
 | `service.loadBalancerIP`                | loadBalancerIP for Kafka Service                                                                                                                          | `nil`                                                   |
 | `service.annotations`                   | Service annotations                                                                                                                                       | ``                                                      |
+| `externalAccess.enabled`               | Enable Kubernetes external cluster access to Kafka brokers                                                                                                 | `false`                                                 |
+| `externalAccess.service.type`          | Kubernetes Servive type for external access. It can be NodePort or LoadBalancer                                                                            | `LoadBalancer`                                          |
+| `externalAccess.service.port`          | Kafka port used for external access when service type is LoadBalancer                                                                                      | `19092`                                                 |
+| `externalAccess.service.loadBalancerIP`| Array of load balancer IPs for Kafka brokers.                                                                                                              | `[]`                                                    |
+| `externalAccess.service.domain`        | Domain or external ip used to configure Kafka external listener when service type is NodePort                                                              | `nil`                                                   |
+| `externalAccess.service.nodePort`      | Array of node ports used to configure Kafka external listener when service type is NodePort                                                                | `[]`                                                    |
```

Comment:
I think this section deserves a paragraph explaining how to use it. Now you'll have two services and it's very confusing:

```
service:
  type: ClusterIP
  port: 9092
  annotations:
...
externalAccess:
  enabled: false
  service:
    type: LoadBalancer
    port: 19092
    loadBalancerIP: []
    # domain: mydomain.com
    nodePort:
```

We should properly explain this.

Reply:
added

Reply:
This is great thanks!